### ISSUE TITLE
[BUG] SFT validation fails when use_sequence_packing is enabled

### ISSUE THREAD

**Issue by XChen-Zero** [alibaba/ROLL#294]
## Description

When `use_sequence_packing` is enabled in SFT training, validation fails during `val_step`.

In the current implementation:

- `train_step` wraps the loss function with `SequencePackingSFTLossWrapper`
- `val_step` directly uses the raw loss function

As a result, when sequence packing is enabled, `forward_step` receives incompatible
inputs and validation fails at runtime.

## Affected Code

`roll/pipeline/sft/sft_worker.py`

## Expected Behavior

Validation should apply the same loss wrapper as training when
`use_sequence_packing` is enabled, so that SFT training and validation work
correctly with sequence packing.

## Additional Context

I have submitted a PR that fixes this issue by applying
`SequencePackingSFTLossWrapper` in `val_step`:

- PR: [XChen-Zero:fix/sft-val-sequence-packing](https://github.com/alibaba/ROLL/pull/293)

**Comment by Schnabel-8** [comment]
Thank you for identifying this issue and submitting the PR—there was indeed a problem in the previous implementation. We recently refactored the sequence packing logic, and validation now works correctly in the SFT pipeline when sequence packing is enabled. Below are the experimental results:

<img width="3132" height="1390" alt="Image" src="https://github.com/user-attachments/assets/ef8da33c-9e63-4589-ab53-1a11a04e89b6" />
